bot/main.go

```diff
@@ -147,9 +147,16 @@ func parseFlags() (flags, error) {
 		return flags{}, trace.BadParameter("reviewers required for assign and check")
 	}
 
-	data, err := base64.StdEncoding.DecodeString(*reviewers)
-	if err != nil {
-		return flags{}, trace.Wrap(err)
+	var decodedReviewers string
+	// Support base64-encoded JSON reviewer string and just JSON without base64 encoding
+	if strings.Contains(*reviewers, "{") {
```

Comment:
Nit: Should we make this check a little more strict and use "starts with {" (i.e. use HasPrefix instead of Contains)?